cuebot/src/test/java/com/imageworks/spcue/test/dao/postgres/LayerDaoTests.java

```diff
@@ -281,7 +281,9 @@ public void testUpdateLayerMinMemory() {
          */
         layerDao.updateLayerMinMemory(layer, 8096);
         LayerDetail l2 = layerDao.findLayerDetail(getJob(), "pass_1");
```

Comment:
Recommendation:

Better read the value from cuebot/src/test/resources/opencue.properties instead of hardcoding. It improves consistency, simplifies updates, keeps the code cleaner, and supports flexible configurations across test scenarios.

Reply:
I don't think this comment was applied to the correct line. I don't see any hardcoded value here.